src/hatti/views/map.cljs

```diff
@@ -329,6 +458,49 @@
                    :on-click (click-fn #(om/update! geofield field))}]
                  (get-label field) [:br]])]]]))))))
 
+(defn map-layer-selector
+  [cursor owner]
+  (reify
+    om/IInitState
+    (init-state [_]
+      {:expanded false
+       :styles ["basic" "streets" "bright" "light" "dark" "satellite"]
```

Comment:
ok cool, and custom basemaps get added onto this list? I'm fine putting these in a `def default-basemaps` for now, until we need to be dynamic with the defaults here, I think that's fine.